CommonCompilerConfig.cmake

```diff
@@ -316,6 +326,7 @@ IF(UNIX AND CMAKE_COMPILER_IS_GNUCXX AND (NOT "${CMAKE_BUILD_TYPE}" STREQUAL "De
     list(APPEND EXTERNAL_STATICLIB_CMAKE_DEFINITIONS_OPTIMIZED
          -DCMAKE_AR:PATH=gcc-ar
          -DCMAKE_RANLIB:PATH=gcc-ranlib)
+    # And ditto for externals that use the configure-based build system
```

Comment:
ditto makes sense if you expand the code and look at the preceding statement's comment in the code.